.github/workflows/test.yml

```diff
@@ -41,15 +41,23 @@ jobs:
         run: ./scripts/format_check.sh
 
   test:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
```

Comment:
For my own understanding, was this the default strategy?

Reply:
The default strategy is to abort all other matrix runs when one fails. In this case, I think it is useful to just get all of the results.

Reply:
Gotcha, agreed.